tests/integration/hive_metastore/test_migrate.py

```diff
@@ -46,6 +46,7 @@ def test_migrate_managed_tables(ws, sql_backend, inventory_schema, make_catalog,
 
     target_table_properties = ws.tables.get(f"{dst_schema.full_name}.{src_managed_table.name}").properties
     assert target_table_properties["upgraded_from"] == src_managed_table.full_name
+    assert target_table_properties["upgraded_from_ws"] == str(ws.get_workspace_id())
```

Comment:
Why did you introduce a constant if you don't use it, like in here? :)

Reply:
Fixed